CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_linkedin_source v0.7.0
+## 🚨 Breaking Changes 🚨
+Due to Linkedin Ads API [change in January 2023](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/recent-changes?view=li-lms-2022-12#january-2023), there have been updates in the Linkedin Ads Fivetran Connector and therefore, updates to this Linkedin package. 
+
+The following fields have been completely deprecated in the `stg_linkedin_ads__creative_history` model ([PR #48](https://github.com/fivetran/dbt_linkedin_source/pull/48)):
+- `type`
+- `call_to_action_label_type`
+- `version_tag`
+
+## Updates
+- The following legacy fields have been updated respectively in the connector and [PR #48](https://github.com/fivetran/ dbt_linkedin_source/pull/48) includes the below modifications:
```

Comment:
This markdown link seems to be malformed. I think it is the extra space that may be trowing it off.
![image](https://user-images.githubusercontent.com/74217849/223154868-64569880-4870-47d9-9c44-08166db8618f.png)